bin/reacli.js

```diff
@@ -5,8 +5,31 @@ import program from "commander"
 import pkgInfo from "pkginfo"
 
 import { validateName, validatePath } from "../lib/utils/validators"
-import { createComponent } from "../lib/core"
+import { createComponent, createHook } from "../lib/core"
 
+const createElement = ({ firstParam = null, secondParam = null, options = [] }) => {
+	// Cmd reacli component <path> creates a component architecture
+
+	const path = pathModule.resolve(secondParam)
+
+	if (validatePath(path) && validateName(path)) {
+		try {
+			switch (firstParam) {
+			case "component":
+				createComponent(path, options)
+				break;
+			case "hook":
+				createHook(path, options)
+				break;
+			default:
```

Comment:
Could be cool to call `program.outputHelp()` in `default`

Reply:
Good Idea :+1: